src/Helper/Settings.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\os2forms_organisation\Helper;
+
+use Drupal\Core\KeyValueStore\KeyValueFactoryInterface;
+use Drupal\Core\KeyValueStore\KeyValueStoreInterface;
+use Drupal\os2forms_organisation\Exception\InvalidSettingException;
+use Drupal\os2forms_organisation\Form\SettingsForm;
+
+/**
+ * General settings for os2forms_organisation.
+ */
+final class Settings {
+  /**
+   * The store.
+   *
+   * @var \Drupal\Core\KeyValueStore\KeyValueStoreInterface
+   */
+  private KeyValueStoreInterface $store;
+
+  /**
+   * The key value collection name.
+   *
+   * @var string
+   */
+  private $collection = 'os2forms_organisation.';
+
+  /**
+   * The setting keys.
+   *
+   * @var array|string[]
+   */
+  private array $keys = [
+    SettingsForm::TEST_MODE,
+    SettingsForm::AUTHORITY_CVR,
+    SettingsForm::CERTIFICATE,
+    SettingsForm::CACHE_EXPIRATION,
+    SettingsForm::ORGANISATION_SERVICE_ENDPOINT_REFERENCE,
+    SettingsForm::ORGANISATION_TEST_LEDER_ROLLE_UUID,
+    SettingsForm::ORGANISATION_PROD_LEDER_ROLLE_UUID,
+  ];
```

Comment:
Use the excellent options resolver instead of this, cf. https://github.com/itk-dev/os2forms_digital_post/blob/feature/DW-277-SF1601/src/Helper/Settings.php#L102-L113.

Reply:
Fixed.